ui/src/app/base/components/FormikFormContent/FormikFormContent.tsx

```diff
@@ -79,7 +79,7 @@ const FormikFormContent = <V,>({
   inline,
   loading,
   onSaveAnalytics = {},
-  onSuccess,
+  onSuccess = () => void 0,
```

Comment:
Any specific reason for `void 0`? In other places I see simply `() => void`.

Reply:
This is just setting a default value for the `onSuccess` parameter; `onSuccess` is optional, but we call it without checking whether anything was passed [here](https://github.com/canonical-web-and-design/maas-ui/blob/71aaa9f9f3b50ab697fcc29ff66cdecc8da42bcf/ui/src/app/base/components/FormikFormContent/FormikFormContent.tsx#L114). Looks neater, but requires this default value.